zerver/migrations/0371_invalid_characters_in_topics.py

```diff
@@ -49,9 +49,9 @@ def fix_topics(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None
                     continue
 
                 # We don't want empty topics for stream messages, so we
-                # use (no topic) if the above clean-up leaves us with an empty string.
+                # use general chat if the above clean-up leaves us with an empty string.
                 if fixed_topic == "":
-                    fixed_topic = "(no topic)"
+                    fixed_topic = "general chat"
```

Comment:
This change is not required. We would instead want to add a new migration to change "no topic" to "general chat" for existing rows.

Reply:
Do you want me to implement this? I have not worked with migrations before so I am not sure what you want me to modify here, could therefore need some help

Reply:
@sofbe please take a look at https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html and ask for help (see https://zulip.readthedocs.io/en/latest/contributing/asking-great-questions.html) as needed.

Reply:
Thanks! I had a look at it and wonder if I need to make any changes to the `zerver/models.py` file to generate a migration file, like add a new field to an existing model or should I make a new model? If not, how is the migration file generated?